pkg/apis/nfd/v1/nodefeaturediscovery_types.go

```diff
@@ -0,0 +1,95 @@
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NodeFeatureDiscoverySpec defines the desired state of NodeFeatureDiscovery
+// +k8s:openapi-gen=true
+type NodeFeatureDiscoverySpec struct {
+	Operand OperandSpec `json:"operand"`
+	Config  ConfigSpec  `json:"nfd-worker"`
```

Comment:
If this is just for the worker config, should we name it `WorkerConfig` (and use`json:"workerConfig"`, respectively)?